ceph-salt-formula/salt/ceph-salt/apply/files/latency.conf

```diff
@@ -0,0 +1,12 @@
+
```

Comment:
Can we include the "managed by ceph-salt" header? See https://github.com/ceph/ceph-salt/blob/master/ceph-salt-formula/salt/ceph-salt/apply/files/chrony.conf.j2#L1